content/news/2023_05_04.en.md

```diff
@@ -0,0 +1,33 @@
+---
+author: "FICO-ITA"
+title: "Are you passionate about Computational Finance and want to obtain a doctorate in this field?"
+date: "2023-05-04"
+description: "Are you passionate about Computational Finance and want to obtain a doctorate in this field?"
+image: "img/noticias/callFICO202301.jpg"
+tags: [
+    "markdown",
+    "css",
+    "html",
+]
+categories: []
+series: []
+aliases: []
+---
+
+The Aeronautics Institute of Technology (ITA), the most prestigious STEM school in Brazil, is open for applications for its Master's and PhD programs until May 5th. 📅
+
+As part of this selection process, we are seeking talented candidates interested in pursuing a doctoral degree in Computational Finance within the Interest Group on Computational Finance and Systematic Investments, FICO - ITA | Employing Science for Finance and Investments .
```

Comment:
Melhor tirar o `| Employing Science for Finance and Investments` no final. Ficou estrado, este texto é apenas para lincar com o site do Linkedin, não é? O ponto final está flutuando também @sbruzzi-ita